// @imagina/qbanner/_config/backendPages.js
export default {
  //Home Page
  index: {
    permission: 'ibanners.positions.index',
    activated: true,
    path: '/banner/index',
    name: 'qbanner.admin.sliders',
    page: () => import('@imagina/qbanner/_pages/admin/index'),
    layout: () => import('@imagina/qsite/_layouts/master.vue'),
    title: 'qbanner.sidebar.adminIndex',
    icon: 'fas fa-photo-video',
    authenticated: true,
  },
  showPosition: {
    permission: 'ibanners.positions.index',
    activated: true,
    path: '/banner/show/:id',
    name: 'qbanner.admin.positions.show',
    page: () => import('@imagina/qbanner/_pages/admin/slider/show.vue'),
    layout: () => import('@imagina/qsite/_layouts/master.vue'),
    title: 'qbanner.sidebar.adminIndex',
    icon: 'fas fa-photo-video',
    authenticated: true,
  },
  createBanner: {
    permission: 'ibanners.banners.index',
    activated: true,
    path: '/banner/item/create/:sliderId',
    name: 'qbanner.admin.banner.create',
    page: () => import('@imagina/qbanner/_pages/admin/slide/create.vue'),
    layout: () => import('@imagina/qsite/_layouts/master.vue'),
    title: 'qbanner.sidebar.adminIndex',
    icon: 'fas fa-photo-video',
    authenticated: true,
  },
  updateBanner: {
    permission: 'ibanners.banners.index',
    activated: true,
    path: '/banner/item/update/:sliderId/:id',
    name: 'qbanner.admin.banner.update',
    page: () => import('@imagina/qbanner/_pages/admin/slide/show.vue'),
    layout: () => import('@imagina/qsite/_layouts/master.vue'),
    title: 'qbanner.sidebar.adminIndex',
    icon: 'fas fa-photo-video',
    authenticated: true,
  },
}
