// @imagina/qsubscription/_i18n/en-us/layout.js
export default {
  newProduct : 'New product',
  updateProduct : 'Update product',
  newPlan : 'New plan',
  updatePlan : 'Update plan',
  newFeature : 'New feature',
  updateFeature : 'Update feature',
  goProducts : 'Go to the list of products',
  form : {
    features : 'Features',
    code : 'Code',
    recommendation : 'Recommendation',
    display_order : 'Display order',
    free : 'Free',
    visible : 'Visible',
    price : 'Price',
    frequency : 'Frequency',
    bill_cycle : "Every",
    bill_cycles : {
      years : "Year(s)",
      months : "Month(s)",
      weeks : "Week(s)"
    },
    trial_period : "Trial period",
    caption:"Caption",
    type:"Type",
    types : {
      text : "text",
      quantity : "quantity",
      boolean : "Boolean"
    },
    unit:"Unit",
    checkout:{
      shippingInformation:"Shipping information",
      cityResidence:"City of residence",
      paymentMethods:"Payment Methods",
      homeDelivery:"Home Delivery",
      storePickup:"Store Pickup",
      products:"Products",
      shippingOptions:"Shipping options",
      finalizePurchase:"Finalize purchase",
      shipping:"Shipping",
      discount:"Discount",
      calculateShipping:"Calculate Shipping",
      processPurchase:"Process purchase",

    }
  }
}
