// @imagina/qmarketplace/_config/backendPages.js
export default {
  categoriesStore: {
    permission: 'qmarketplace.productcategories.manage',
    activated: true,
    path: '/qmarketplace/categories/store',
    name: 'qmarketplace.admin.categories.store.index',
    page: () => import('@imagina/qmarketplace/_layouts/admin/productCategories/index'),
    layout: () => import('src/layouts/admin'),
    title: 'qcommerce.sidebar.adminCategories',
    icon: 'fas fa-layer-group',
    authenticated: true,
  },
  // Super admin Stores
  themes: {
    permission: 'marketplace.themes.manage',
    activated: true,//
    path: '/qmarketplace/themes/index',
    name: 'qmarketplace.admin.themes.index',//
    page: () => import('@imagina/qmarketplace/_layouts/admin/themes/index'),
    layout: () => import('src/layouts/admin'),
    title: 'qmarketplace.sidebar.adminThemes',
    icon: 'fas fa-store',
    authenticated: true
  },
  categories: {//
    permission: 'marketplace.categorystores.manage',
    activated: true,//
    path: '/qmarketplace/categories/index',
    name: 'qmarketplace.admin.categories.index',
    page: () => import('@imagina/qmarketplace/_layouts/admin/categories/index'),
    layout: () => import('src/layouts/admin'),
    title: 'qmarketplace.sidebar.adminCategories',
    icon: 'fas fa-store',
    authenticated: true
  },
  stores: {
    permission: 'marketplace.stores.manage',
    activated: true,
    path: '/qmarketplace/stores/index',
    name: 'qmarketplace.admin.stores.index',
    page: () => import('@imagina/qmarketplace/_layouts/admin/stores/index'),
    layout: () => import('src/layouts/admin'),
    title: 'qmarketplace.sidebar.adminMarketplace',
    icon: 'fas fa-store',
    authenticated: true
  },
  storesCreate:{
    activated: true,
    path: '/qmarketplace/stores/create',
    name: 'qmarketplace.admin.stores.create',
    page: () => import('@imagina/qmarketplace/_layouts/admin/stores/form'),
    layout: () => import('src/layouts/admin'),
    title: 'qmarketplace.sidebar.adminMarketplace',
    icon: 'fas fa-store',
    authenticated: true,
  },
  storesEdit:{
    activated: true,
    path: '/qmarketplace/stores/edit/:id',
    name: 'qmarketplace.admin.stores.edit',
    page: () => import('@imagina/qmarketplace/_layouts/admin/stores/form'),
    layout: () => import('src/layouts/admin'),
    title: 'qmarketplace.sidebar.adminMarketplace',
    icon: 'fas fa-store',
    authenticated: true,
  },
  storesTheme: {
    permission: 'marketplace.themes.manage',
    activated: true,
    path: '/qmarketplace/theme/store/:id',
    name: 'qmarketplace.admin.theme.store.index',
    page: () => import('@imagina/qmarketplace/_layouts/admin/stores/theme/index'),
    layout: () => import('src/layouts/admin'),
    title: 'qmarketplace.sidebar.adminTheme',
    icon: 'fas fa-store',
    authenticated: true
  },
  storeProducts: {
     permission: 'marketplace.stores.manage',
    activated: true,
    path: '/qmarketplace/products/store/:id',
    name: 'qmarketplace.admin.products.store.index',
    page: () => import('@imagina/qmarketplace/_layouts/admin/stores/products/index'),
    layout: () => import('src/layouts/admin'),
    title: 'qmarketplace.sidebar.adminProducts',
    icon: 'fas fa-store',
    authenticated: true
  },
  storeProductsCreate: {
    permission: 'marketplace.stores.manage',
    activated: true,
    path: '/qmarketplace/products/create/:storeId',
    name: 'qmarketplace.admin.products.store.create',
    page: () => import('@imagina/qmarketplace/_layouts/admin/stores/products/form'),
    layout: () => import('src/layouts/admin'),
    title: 'qmarketplace.sidebar.adminProducts',
    icon: 'fas fa-store',
    authenticated: true
  },
  //end Super admin Sores
  //Edit data of selected store
  myStore:{
    permission: 'marketplace.stores.mystore',
    activated: true,
    path: '/admin/stores/my-store',
    name: 'qmarketplace.admin.stores.my.store',
    page: () => import('@imagina/qmarketplace/_layouts/admin/stores/formMyStore'),
    layout: () => import('src/layouts/admin'),
    title: 'qmarketplace.sidebar.business.myStore',
    icon: 'fas fa-store',
    authenticated: true,
},

//Edit theme and colors of selected store
  myStoreTheme: {
    permission: 'marketplace.stores.mystore',
    activated: true,
    path: '/admin/stores/my-store/theme',
    name: 'qmarketplace.admin.stores.my.store.theme',
    page: () => import('@imagina/qmarketplace/_layouts/admin/stores/theme/formStoreTheme'),
    layout: () => import('src/layouts/admin'),
    title: 'qmarketplace.sidebar.adminTheme',
    icon: 'fas fa-palette',
    authenticated: true
  },

  myStoreContacts: {
    permission: 'marketplace.stores.mystore',
    activated: true,
    path: '/admin/stores/my-store/contact-messages/index',
    name: 'qmarketplace.admin.storecontacts.index',
    page: () => import('@imagina/qmarketplace/_layouts/admin/storeContacts/index'),
    layout: () => import('src/layouts/admin'),
    title: 'qmarketplace.sidebar.adminStoreContacts',
    icon: 'fas fa-store',
    authenticated: true
  },

  //Products of selected store
  myStoreProducts: {
    permission: 'marketplace.products.index',
    activated: true,
    path: '/admin/stores/my-store/products',
    name: 'qmarketplace.admin.stores.my.store.products',
    page: () => import('@imagina/qmarketplace/_layouts/admin/stores/products/indexStoreProducts'),
    layout: () => import('src/layouts/admin'),
    title: 'qmarketplace.sidebar.business.products',
    icon: 'fas fa-store',
    authenticated: true
  },
  //Create products of selected store
  myStoreProductCreate: {
    permission: 'marketplace.products.create',
    activated: true,
    path: '/admin/stores/my-store/products/create',
    name: 'qmarketplace.admin.stores.business.products.create',
    page: () => import('@imagina/qmarketplace/_layouts/admin/stores/products/formMyStore'),
    layout: () => import('src/layouts/admin'),
    title: 'qmarketplace.sidebar.business.products',
    icon: 'fas fa-store',
    authenticated: true
  },
  //Edit product of selected store
  myStoreProductEdit: {
    permission: 'marketplace.products.edit',
    activated: true,
    path: '/admin/stores/my-store/products/edit/:id',
    name: 'qmarketplace.admin.stores.business.products.edit',
    page: () => import('@imagina/qmarketplace/_layouts/admin/stores/products/formMyStore'),
    layout: () => import('src/layouts/admin'),
    title: 'qmarketplace.sidebar.business.products',
    icon: 'fas fa-store',
    authenticated: true
  },

  //Product categories of selected store
  myStoreCategoryProducts: {
    permission: 'marketplace.productcategories.index',
    activated: true,
    path: '/admin/stores/my-store/product/category',
    name: 'qmarketplace.admin.stores.myStore.productsCategory',
    page: () => import('@imagina/qmarketplace/_layouts/admin/productCategories/index'),
    layout: () => import('src/layouts/admin'),
    title: 'qmarketplace.sidebar.business.categories',
    icon: 'fas fa-store',
    authenticated: true
  },
  myStoreOrders: {
    permission: 'marketplace.orders.index',
    activated: true,
    path: '/admin/stores/my-store/orders',
    name: 'qmarketplace.admin.shipping.orders.index',
    page: () => import('@imagina/qmarketplace/_layouts/admin/order/index'),
    layout: () => import('src/layouts/admin'),
    title: 'qmarketplace.sidebar.business.orders',
    icon: 'fas fa-box-open',
    authenticated: true,
  },
  order: {
    permission: 'marketplace.orders.edit',
    activated: true,
    path: '/admin/stores/my-store/order/:id',
    name: 'qmarketplace.admin.shipping.orders.show',
    page: () => import('@imagina/qmarketplace/_layouts/admin/order/show'),
    layout: () => import('src/layouts/admin'),
    title: 'qmarketplace.sidebar.business.order',
    icon: 'fas fa-box-open',
    authenticated: true,
  },
  myStorePolls: {
    permission: 'marketplace.polls.index',
    activated: true,
    path: '/admin/stores/my-store/polls/index',
    name: 'qmarketplace.admin.polls.index',
    page: () => import('@imagina/qmarketplace/_layouts/admin/quiz/polls/index'),
    layout: () => import('src/layouts/admin'),
    title: 'qquiz.sidebar.adminPolls',
    icon: 'fas fa-poll',
    authenticated: true
  },
  myStoreQuestions: {
    permission: 'marketplace.questions.index',
    activated: true,
    path: '/admin/stores/my-store/poll/:id/questions/index',
    name: 'qmarketplace.admin.questions.index',
    page: () => import('@imagina/qmarketplace/_layouts/admin/quiz/questions/index'),
    layout: () => import('src/layouts/admin'),
    title: 'qquiz.sidebar.adminQuestions',
    icon: 'fas fa-question-circle',
    authenticated: true
  },
  myStoreQuestionReport: {
    permission: 'marketplace.questions.index',
    activated: true,
    path: '/admin/stores/my-store/question/:id/report',
    name: 'qmarketplace.admin.questions.report.index',
    page: () => import('@imagina/qmarketplace/_layouts/admin/quiz/questions/report'),
    layout: () => import('src/layouts/admin'),
    title: 'qquiz.sidebar.adminQuestions',
    icon: 'fas fa-question-circle',
    authenticated: true
  },
  myStoreAnswers: {
    permission: 'marketplace.answers.index',
    activated: true,
    path: '/admin/stores/my-store/poll/:poll_id/question/:id/answers/index',
    name: 'qmarketplace.admin.answers.index',
    page: () => import('@imagina/qmarketplace/_layouts/admin/quiz/answers/index'),
    layout: () => import('src/layouts/admin'),
    title: 'qquiz.sidebar.adminAnswers',
    icon: 'fas fa-question-circle',
    authenticated: true
  },
  favoriteUsers: {
    permission: 'marketplace.stores.mystore',
    activated: true,
    path: '/admin/followers/index',
    name: 'qmarketplace.admin.favoritestores.index',
    page: () => import('@imagina/qmarketplace/_layouts/admin/stores/favoritestores/index'),
    layout: () => import('src/layouts/admin'),
    title: 'qmarketplace.sidebar.adminFavoritestores',
    icon: 'fas fa-users',
    authenticated: true
  },
  coupons: {
    permission: 'marketplace.coupons.index',
    activated: true,
    path: '/admin/stores/my-store/coupons',
    name: 'qmarketplace.admin.coupons.index',
    page: () => import('@imagina/qmarketplace/_layouts/admin/coupons/index'),
    layout: () => import('src/layouts/admin'),
    title: 'qcommerce.sidebar.adminCoupons',
    icon: 'fas fa-tags',
    authenticated: true,
  },
  couponsCreate: {
    permission: 'marketplace.coupons.create',
    activated: true,
    path: '/admin/stores/my-store/coupons/create',
    name: 'qmarketplace.admin.coupons.create',
    page: () => import('@imagina/qmarketplace/_layouts/admin/coupons/form'),
    layout: () => import('src/layouts/admin'),
    title: 'qcommerce.sidebar.createCoupons',
    icon: 'fas fa-ticket-alt',
    authenticated: true,
  },
  couponsEdit: {
    permission: 'marketplace.coupons.edit',
    activated: true,
    path: '/admin/stores/my-store/coupons/:id',
    name: 'qmarketplace.admin.coupons.edit',
    page: () => import('@imagina/qmarketplace/_layouts/admin/coupons/form'),
    layout: () => import('src/layouts/admin'),
    title: 'qcommerce.sidebar.editCoupons',
    icon: 'fas fa-ticket-alt',
    authenticated: true,
  },
  myChatMessages: {
    permission: 'marketplace.chat.index',
    activated: true,
    path: '/admin/stores/my-store/chat/messages',
    name: 'qchat.admin.messages.index',
    page: () => import('@imagina/qchat/_pages/admin/index'),
    layout: () => import('src/layouts/admin'),
    title: 'qchat.sidebar.adminMessages',
    icon: 'far fa-comments',
  },
  myNotifications: {
    permission: 'marketplace.notification.index',
    activated: true,
    path: '/admin/stores/my-store/notifications',
    name: 'qmarketplace.admin.notifications.index',
    page: () => import('@imagina/qmarketplace/_layouts/admin/notification/notifications'),
    layout: () => import('src/layouts/admin'),
    title: 'qnotification.sidebar.notification',
    icon: 'fas fa-bell'
  },
  myBannersIndex: {
    permission: 'marketplace.bannerpositions.index',
    activated: true,
    path: '/admin/stores/my-store/banner/index',
    name: 'qmarketplace.admin.banners.positions',
    page: () => import('@imagina/qmarketplace/_layouts/admin/banners/index'),
    layout: () => import('src/layouts/admin'),
    title: 'qbanner.sidebar.adminIndex',
    icon: 'fas fa-photo-video',
    authenticated: true,
  },
  myBannersShowPosition: {
    permission: 'marketplace.bannerpositions.edit',
    activated: true,
    path: '/admin/stores/my-store/banner/show/:id',
    name: 'qmarketplace.admin.banners.positions.show',
    page: () => import('@imagina/qmarketplace/_layouts/admin/banners/slider/show.vue'),
    layout: () => import('src/layouts/admin'),
    title: 'qbanner.sidebar.adminIndex',
    icon: 'fas fa-photo-video',
    authenticated: true,
  },
  myBannersCreateBanner: {
    permission: 'marketplace.banners.index',
    activated: true,
    path: '/admin/stores/my-store/banner/item/create/:sliderId',
    name: 'qmarketplace.admin.banners.banner.create',
    page: () => import('@imagina/qmarketplace/_layouts/admin/banners/slide/create.vue'),
    layout: () => import('src/layouts/admin'),
    title: 'qbanner.sidebar.adminIndex',
    icon: 'fas fa-photo-video',
    authenticated: true,
  },
  myBannersUpdateBanner: {
    permission: 'marketplace.banners.index',
    activated: true,
    path: '/admin/stores/my-store/banner/item/update/:sliderId/:id',
    name: 'qqmarketplace.admin.banners.banner.update',
    page: () => import('@imagina/qmarketplace/_layouts/admin/banners/slide/show.vue'),
    layout: () => import('src/layouts/admin'),
    title: 'qbanner.sidebar.adminIndex',
    icon: 'fas fa-photo-video',
    authenticated: true,
  },
  mySubscription: {
    permission: 'marketplace.stores.mystore',
    activated: true,
    path: '/admin/stores/my-subscription',
    name: 'qmarketplace.admin.subscriptions.mysubscriptions',
    page: () => import('@imagina/qmarketplace/_layouts/admin/subscriptions/subscription.vue'),
    layout: () => import('src/layouts/admin'),
    title: 'qmarketplace.sidebar.business.mySubscription',
    icon: 'fas fa-photo-video',
    authenticated: true,
  },
  benefits: {
    //permission: null,
    permission: 'marketplace.leveltypes.index',
    activated: true,
    path: '/admin/benefits',
    name: 'qmarketplace.admin.benefits.index',
    page: () => import('@imagina/qmarketplace/_layouts/admin/benefits/index'),
    layout: () => import('src/layouts/admin'),
    title: 'qmarketplace.sidebar.adminBenefits',
    icon: 'fas fa-ticket-alt',
    authenticated: true,
  },
  levelTypes: {//
    //permission: null,
    permission: 'marketplace.leveltypes.index',
    activated: true,
    path: '/admin/level-types',
    name: 'qmarketplace.admin.level.types.index',
    page: () => import('@imagina/qmarketplace/_layouts/admin/levelTypes/index'),
    layout: () => import('src/layouts/admin'),
    title: 'qmarketplace.sidebar.adminLevelTypes',
    icon: 'fas fa-ticket-alt',
    authenticated: true,
  },
  levelsCriteria: {
    //permission: null,
    permission: 'marketplace.levelcriterias.index',
    activated: true,
    path: '/admin/level-criterias',
    name: 'qmarketplace.admin.levels.criteria.index',
    page: () => import('@imagina/qmarketplace/_layouts/admin/levelsCriteria/index'),
    layout: () => import('src/layouts/admin'),
    title: 'qmarketplace.sidebar.adminLevelCriterias',
    icon: 'fas fa-ticket-alt',
    authenticated: true,
  },
  levels: {
    permission: 'marketplace.levels.manage',
    activated: true,
    path: '/admin/levels',
    name: 'qmarketplace.admin.levels.index',
    page: () => import('@imagina/qmarketplace/_layouts/admin/levels/index'),
    layout: () => import('src/layouts/admin'),
    title: 'qmarketplace.sidebar.adminLevels',
    icon: 'fas fa-ticket-alt',
    authenticated: true,
  },
  levelsCreate: {
    permission: 'marketplace.levels.create',
    activated: true,
    path: '/admin/levels/create',
    name: 'qmarketplace.admin.levels.create',
    page: () => import('@imagina/qmarketplace/_layouts/admin/levels/form.vue'),
    layout: () => import('src/layouts/admin'),
    title: 'qmarketplace.sidebar.adminLevels',
    icon: 'fas fa-photo-video',
    authenticated: true,
  },
  levelsEdit: {
    permission: 'marketplace.levels.edit',
    activated: true,
    path: '/admin/levels/update/:id',
    name: 'qmarketplace.admin.levels.update',
    page: () => import('@imagina/qmarketplace/_layouts/admin/levels/form.vue'),
    layout: () => import('src/layouts/admin'),
    title: 'qmarketplace.sidebar.adminLevels',
    icon: 'fas fa-photo-video',
    authenticated: true,
  },
  myStoreReport:{
    permission: 'marketplace.stores.mystore',
    activated: true,
    path: '/admin/stores/my-store/report',
    name: 'qmarketplace.admin.mystore.report',
    page: () => import('@imagina/qmarketplace/_layouts/admin/stores/report/index.vue'),
    layout: () => import('src/layouts/admin'),
    title: 'qmarketplace.sidebar.business.storeReport',
    icon: 'fas fa-file-alt',
    authenticated: true,
  },
  myStoreLevel:{
    permission: 'marketplace.stores.mystore',
    activated: true,
    path: '/admin/stores/my-store/level',
    name: 'qmarketplace.admin.mystore.level',
    page: () => import('@imagina/qmarketplace/_layouts/admin/stores/levels/index.vue'),
    layout: () => import('src/layouts/admin'),
    title: 'qmarketplace.sidebar.business.storeLevel',
    icon: 'fas fa-sign-out-alt',
    authenticated: true,
  },
  myStoreServices:{
    permission: 'marketplace.posts.index',
    activated: true,
    path: '/admin/stores/my-store/services',
    name: 'qmarketplace.admin.mystore.services',
    page: () => import('@imagina/qmarketplace/_layouts/admin/blog/posts/index.vue'),
    layout: () => import('src/layouts/admin'),
    title: 'qmarketplace.sidebar.business.services',
    icon: 'fas fa-sign-out-alt',
    authenticated: true,
  },
  myStoreServicesCategory:{
    permission: 'marketplace.postcategories.index',
    activated: true,
    path: '/admin/stores/my-store/services/category',
    name: 'qmarketplace.admin.mystore.services.categories',
    page: () => import('@imagina/qmarketplace/_layouts/admin/blog/categories/index.vue'),
    layout: () => import('src/layouts/admin'),
    title: 'qmarketplace.sidebar.business.servicesCategory',
    icon: 'fas fa-sign-out-alt',
    authenticated: true,
  },
  myStoreLogout:{
    permission: null,
    activated: true,
    path: '/auth/logout',
    name: 'qmarketplace.admin.mystore.logout',
    title: 'quser.layout.label.logout',
    icon: 'fas fa-sign-out-alt',
    authenticated: true,
  }
  //create
}
