// @imagina/qsubscription/_i18n/es/layout.js
export default {
  newProduct : 'Nuevo producto',
  updateProduct : 'Actualizar producto',
  newPlan : 'Nuevo plan',
  updatePlan : 'Actualizar Plan',
  newFeature : 'Nueva característica',
  updateFeature : 'Actualizar característica',
  goProducts : 'Ir al listado de productos',
  form : {
    features : 'Características',
    code : 'Código',
    recommendation : 'Recomendación',
    display_order : 'Orden de Visualización',
    free : 'Gratis',
    visible : 'Visible',
    price : 'Precio',
    frequency : 'Frecuencia',
    bill_cycle : "Cada",
    bill_cycles : {
      years : "Año(s)",
      months : "Mes(es)",
      weeks : "Semana(s)"
    },
    trial_period : "Período de prueba",
    caption:"Subtítulo",
    type:"Tipo",
    types : {
      text : "Texto",
      quantity : "Cantidad",
      boolean : "Booleano"
    },
    unit:"Unidad",
    checkout:{
      shippingInformation:"Datos de envío",
      cityResidence:"Ciudad de residencia",
      paymentMethods:"Métodos de Pago",
      homeDelivery:"Envío a Domicilio",
      storePickup:"Recoger en Tienda",
      products:"Productos",
      shippingOptions:"Opciones de envio",
      finalizePurchase:"Finalizar compra",
      shipping:"Envío",
      discount:"Descuento",
      calculateShipping:"Calcular Envío",
      processPurchase:"Procesar Compra",

    },
    paymentMethods:{
      apiKey : 'Codigo API',
      apiLogin : 'API Inicio de sesión',
      transactionKey : 'Codigo de transacción',
      clientKey : 'Codigo de cliente',
      clientSecret : 'Codigo Secreto',
      merchantId : 'ID Comerciante',
      mode : 'Modo',
      accountId : 'ID Cuenta',
      allowTestCreditCard : 'Tarjeta de credito de prueba',
      updatePaymentMethod : 'Actualizar metodo de pago',
    }
    
  }
}
